sklearn/ensemble/_weight_boosting.py

```diff
@@ -482,6 +482,11 @@ class AdaBoostClassifier(
     For a detailed example of using AdaBoost to fit a sequence of DecisionTrees
     as weaklearners, please refer to
     :ref:`sphx_glr_auto_examples_ensemble_plot_adaboost_multiclass.py`.
+
+    A detailed example of using AdaBoost to fit a non-linear seperable
+    classification dataset composed of two Guassian Quantiles Clusters
+    and plot decision boundary and decision scores, please refer to
+    :ref:`sphx_glr_auto_examples_ensemble_plot_adaboost_twoclass.py`
```

Comment:
```suggestion

    For a detailed example of using AdaBoost to fit a non-linearly seperable
    classification dataset composed of two Gaussian quantiles clusters, please
    refer to :ref:`sphx_glr_auto_examples_ensemble_plot_adaboost_twoclass.py`.
```

Reply:
Suggested changes has been made and committed it.